metrics/web/lib/common/presentation/metrics_theme/config/dimensions_config.dart

```diff
@@ -13,11 +13,11 @@ class DimensionsConfig {
   static const double buttonHeight = 48.0;
 
   /// An outer diameter of the graph indicator.
-  static const double graphIndicatorOuterDiameter = 5.33;
+  static const double graphIndicatorOuterDiameter = 7.0;
 
   /// An inner diameter of the graph indicator.
-  static const double graphIndicatorInnerDiameter = 2.13;
+  static const double graphIndicatorInnerDiameter = 2.8;
```

Comment:
It seems like the inner diameter is equal to `4.0`.
![image](https://user-images.githubusercontent.com/70939496/108199118-357c4f80-7125-11eb-8231-e6da1e98e099.png)